rest_tools/server/handler.py

```diff
@@ -97,7 +98,7 @@ def __init__(self, *args, **kwargs) -> None:
         try:
             super().__init__(*args, **kwargs)
         except Exception:
-            logging.error('error', exc_info=True)
+            logger.error('error', exc_info=True)
```

Comment:
To be consistent, did you want to change this to `LOGGER`?

Reply:
sure thing